tests/foreman/ui/test_subscription.py

```diff
@@ -190,6 +190,49 @@ def test_positive_access_with_non_admin_user_with_manifest(test_name):
                 == DEFAULT_SUBSCRIPTION_NAME)
 
 
+@tier2
+def test_positive_access_manifest_as_another_admin_user(test_name):
+    """Other admin users should be able to access and manage a manifest
+    uploaded by a different admin.
+
+    :id: 02e319da-3b7a-4694-9164-475c2c71b9a8
+
+    :expectedresults: Other admin user should see/manage the manifest
+
+    :BZ: 1669241
+
+    :customerscenario: true
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+    """
+    org = entities.Organization().create()
+    user1_password = gen_string('alphanumeric')
+    user1 = entities.User(
+        admin=True,
+        password=user1_password,
+        organization=[org],
+        default_organization=org,
+    ).create()
+    user2_password = gen_string('alphanumeric')
+    user2 = entities.User(
+        admin=True,
+        password=user2_password,
+        organization=[org],
+        default_organization=org,
+    ).create()
+    # use the first admin to upload a manifest
+    with Session(test_name, user=user1.login, password=user1_password) as session:
+        manifests.upload_manifest_locked(org.id)
+        assert session.subscription.has_manifest
+    # try to view the manifest with another admin
+    with Session(test_name, user=user2.login, password=user2_password) as session:
+        assert session.subscription.has_manifest
+        session.subscription.delete_manifest(ignore_error_messages=['404 Not Found'])
+        assert not session.subscription.has_manifest
```

Comment:
Would it be useful to also add assertion that checks if specific product is available? So we can have little more confidence that this particular manifest is visible, and not some random unrelated one?

Reply:
That's not strictly necessary, as they're both in the same organization, but I can certainly add a check.